admin_manual/configuration/occ_command.rst

```diff
@@ -0,0 +1,252 @@
+=====================
+Using the occ Command
+=====================
+
+ownCloud's ``occ`` command (ownCloud console) is ownCloud's command-line 
+interface. You can perform many common server operations with ``occ``::
+
+* Maintenance tasks
+* Manage apps
+* Upgrade the ownCloud database
+* Reset passwords, including administrator passwords
+* List files owned by users
+* Convert the ownCloud database from SQLite to a more performant DB
+* Query and change LDAP settings
+
```

Comment:
Maybe add a notice that depending on the installed applications more or less features might be available?